src/cargo/core/compiler/fingerprint/mod.rs

```diff
@@ -405,11 +405,6 @@ pub fn prepare_target(
     unit: &Unit,
     force: bool,
 ) -> CargoResult<Job> {
-    let _p = profile::start(format!(
```

Comment:
The old profiler records `PackageId` and target name, while the new profiler misses that information as it skips the entire `unit` arg. Should we re-include that?

Reply:
See #13538